js/app.js

```diff
@@ -32,7 +56,7 @@ function question1(){
   }
 }
 
-function question2(){
+function question2(answer2){
```

Comment:
It's also a good idea to name the variables you use in your function definitions differently than your global variables. Inside this function you have the line `if (answer2 === 'yes' || answer2 === 'y')` do you know if answer2 is pulling from the variable you insert when you call this function, or the global variable?

Reply:
I fixed this in my later version of the code. I Think